README.md

```diff
@@ -95,14 +95,23 @@ class Record:
     id: str
     seq: str
     qual: Optional[str]
-
-    def is_fasta(self) -> bool
-    def is_fastq(self) -> bool
-    def normalize(self, iupac: bool)
+    def __init__(self, id: str, seq: str, qual: Optional[str]) -> None: ...
+    @property
```

Comment:
IMO it would be clearer to show these methods with examples than to continue to use the "shape" style docs in the current README